nle/scripts/play.py

```diff
@@ -116,15 +117,13 @@ def play():
                 print("-" * 8)
                 print(obs["blstats"])
                 if not FLAGS.print_frames_separately:
-                    print("\033[33A")  # Go up 33 lines.
+                    print("\033[33A")  # Go back up.
```

Comment:
Useful to keep the comment (and maybe even str.fromat the 33) so anyone hacking know which bitto edit to adjust that. I've done so a lot in past. 
```suggestion
                    print("\033[%dA" % (33))  # Go back up 33 lines.
```

Reply:
Happy to splice the `33` in like you suggest, but this is the comment that is just bound to go out of sync, which is extra bad here since there's also the `\033` here ...

Reply:
Moved to make it more expressive by introducing a suggestive function.